dockers/docker-teamd/start.sh

```diff
@@ -1,26 +1,24 @@
 #!/bin/bash
 
+. config.sh
+
 TEAMD_CONF_PATH=/etc/teamd
 
-# Before teamd could automatically add newly created host interfaces into the
-# LAG, this workaround will be needed. It will remove the obsolete files and
-# net devices that are failed to be removed in the previous run.
 function start_app {
-    # Remove *.pid and *.sock files if there are any
-    rm -f /var/run/teamd/*
     if [ -d $TEAMD_CONF_PATH ]; then
         for f in $TEAMD_CONF_PATH/*; do
-            # Remove netdevs if there are any
-            intf=`echo $f | awk -F'[/.]' '{print $4}'`
-            ip link del $intf
             teamd -f $f -d
         done
     fi
     teamsyncd &
 }
 
 function clean_up {
-    pkill -9 teamd
+    if [ -d $TEAMD_CONF_PATH ]; then
+        for f in $TEAMD_CONF_PATH/*; do
+            teamd -f $f -k
```

Comment:
So I believe this will successfully stop all the team instances without leaving anything that needs to be manually cleaned up after the next docker starts?